src/admin/site_environments.rst

```diff
@@ -0,0 +1,146 @@
+Site Environments
+=================
+
+The site environments required for running PanDA jobs.
+
+Export LSST_LOCAL_PROLOG
+------------------------
+
+For running PanDA jobs, the site only needs to export LSST_LOCAL_PROLOG.
+When LSST_LOCAL_PROLOG is defined, lsst pilot wrapper will source
+LSST_LOCAL_PROLOG during running. In this way, it will not affect other experiments.
+LSST_LOCAL_PROLOG can be defined in .bashrc. for example::
+
+  export LSST_LOCAL_PROLOG=/sdf/home/l/lsstsvc1/.lsst/local_prolog.sh
+
+The content of LSST_LOCAL_PROLOG
+--------------------------------
+
+Here are the content of LSST_LOCAL_PROLOG and corresponding descriptions.
+
+- Timezone:
+
+  To set the timezone, it's good for logging format for jobs from different sites::
```

Comment:
Wording:
Set the timezone to ensure that it is consistent in logs for jobs run across sites in multiple timezones.